server/controllers/refer/oasysConfirmationController.ts

```diff
@@ -0,0 +1,54 @@
+import type { Request, Response, TypedRequestHandler } from 'express'
```

Comment:
The file and class names aren't plural but I guess that's okay? Same with the view

Reply:
Although I tend to prefer using plurals here, something about `oasysConfirmations` feels wrong to me. Going to keep it singular for now, and I think it lines up ok with the `reasonController` in #199